src/WebJobs.Script.WebHost/Security/Authorization/AuthUtility.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.WebJobs.Script.WebHost.Authentication;
+using Microsoft.Azure.WebJobs.Script.WebHost.Security.Authentication;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Security.Authorization
+{
+    public class AuthUtility
+    {
+        public static IEnumerable<string> DefaultAuthorizationSchemes => new[]
+        {
+            AuthLevelAuthenticationDefaults.AuthenticationScheme,
+            JwtBearerDefaults.AuthenticationScheme
+        };
+
+        public static AuthorizationPolicy CreateFunctionPolicy(IEnumerable<string> schemes = null)
+        {
+            schemes = schemes ?? DefaultAuthorizationSchemes;
+            var requirements = new[] { new FunctionAuthorizationRequirement() };
+            return new AuthorizationPolicy(requirements, schemes);
+        }
+
+        public static bool PrincipalHasAuthLevelClaim(ClaimsPrincipal principal, AuthorizationLevel requiredLevel, string keyName = null)
+        {
+            // If the required auth level is anonymous, the requirement is met
+            if (requiredLevel == AuthorizationLevel.Anonymous)
```

Comment:
The previous logic (AuthorizationLevelAttribute.HasAuthorizationLevel) also checked to see if auth was disabled at the host level.

Reply:
That is no longer required. The CLI (or other hosts that may require similar functionality) injects it's own authentication handler, so we don't have to perform those checks in the runtime.

Reply:
If you're saying that WebHostSettings.IsAuthDisabled is no longer used anymore by CLI, we should remove it and the corresponding tests that we have?